### PR TITLE
Documentation for proposed TP-Link Omada integration

### PR DESCRIPTION
**Pull request by MarkGodwin**
## Proposed change

New TP-Link Omada SDN integration needs documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [Integration PR](https://github.com/home-assistant/core/pull/81223)
- Link to parent pull request in the Brands repository: [Brand PR](https://github.com/home-assistant/brands/pull/3824)
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tplink_omada.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: TP-Link Omada SDN Controller
+description: Instructions on integrating TP-Link Omada SDN networking devices to Home Assistant.
+ha_category:
+  - Hub
+ha_release: '2023.03'
```
Comment (frenck) [inline source/_integrations/tplink_omada.markdown]:
```suggestion
ha_release: 2023.3
```

File: source/_integrations/tplink_omada.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@MarkGodwin'
+ha_domain: tplink_omada
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+The `tplink_omada` integration allows you to control your [TP-Link Omada SDN Devices](https://www.tp-link.com/us/omada-sdn/) such as network switches, access points and internet gateways. It requires a local Omada controller (software or hardware controllers), and does not currently support the paid-for TP-Link Omada Cloud Management SAAS Platform.
```
Comment (frenck) [inline source/_integrations/tplink_omada.markdown]:
```suggestion
The TP-Link Omada SDN Controller integration allows you to control your [TP-Link Omada SDN Devices](https://www.tp-link.com/us/omada-sdn/) such as network switches, access points, and internet gateways. It requires a local Omada controller (software or hardware controllers) and does not currently support the paid-for TP-Link Omada Cloud Management SAAS Platform.
```

File: source/_integrations/tplink_omada.markdown [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+The integration provides basic configuration and status of Omada devices controlled by the controller. All supported devices connected to the controller will be added to Home Assistant.
+
+## Configuration UI
+
+- Host: Enter the URL of the Omada management interface.
+- Verify SSL Certificates: Enable to check the validity of the certificate presented by the Omada controller.
+- Username/Password: A user account with permissions to view & configure the site is required.
+
+### Multiple Sites
+If you have multiple sites managed by your controller, you will be prompted to choose which site to manage.
```
Comment (frenck) [inline source/_integrations/tplink_omada.markdown]: Move this below the config flow include (remove the "Configuration UI" header)

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tplink_omada.markdown (hunk 3 above)
Comment (MarkGodwin) [reply source/_integrations/tplink_omada.markdown]: Done. Thanks

**Review by frenck — APPROVED** [review]
Thanks, @MarkGodwin 👍

**Pull request merged**